scripts/generate.sh

```diff
@@ -12,6 +12,6 @@ end=`date +%s`
 runtime=$((end-start))
 echo "Generated endpoints/clients : +$runtime"
 
-bash ./codegen/runner/post-steps.sh "$PREFIX/build"
+bash ./codegen/runner/post-steps.sh "$PREFIX/build" "$PREFIX"
```

Comment:
mm ... seems your $1 == $2 + /build

Reply:
It is coincidental, the post-steps.sh does not assume any relationship between the two params.